### PR TITLE
Additional fixes/enhancements for dashboard example

### PR DESCRIPTION
**Pull request by brendancol**
- lru caching for aggregate creation
- small fix to default value from spreading

### CODE REVIEW

File: examples/dashboard/dashboard.py [hunk 1]
```diff
@@ -330,6 +249,49 @@ def load_datasets(self,outofcore):
         else:
             raise IOError("Unknown data file type; .csv and .castra currently supported")
 
+
+    @memoized_method(maxsize=6)
+    def create_aggregate(self, plot_width, plot_height, x_range, y_range,
+                         agg_field, x_field, y_field, agg_func=None):
+
+        print(">>>>>>>>>>>>>>>>>> RUNNING THE AGGREGATE!!!!!!! <<<<<<<<<<<<<<<<<<")
```
Comment (jbednar) [inline examples/dashboard/dashboard.py]: Presumably not meant to be committed?

File: examples/README.md [hunk 2]
```diff
@@ -31,6 +33,13 @@ integrated with a datashading pipeline. To start, run:
 python dashboard/dashboard.py --config dashboard/nyc_taxi.yml
 ```
 
+Also available, is the [2010 Census Synthetic People dataset in `castra` format](http://s3.amazonaws.com/bokeh_data/census.castra.tar.gz)
+
+This dataset can be visualized in the dashboard example using:
```
Comment (jbednar) [inline examples/README.md]: After unpacking this file in example/data/, this dataset can be visualized in the dashboard example using:

?

File: examples/README.md [hunk 3]
```diff
@@ -31,6 +33,13 @@ integrated with a datashading pipeline. To start, run:
...
+```
+python dashboard/dashboard.py --config dashboard/census.yml
+```
+
```
Comment (jbednar) [inline examples/README.md]: "It's also required for the census.ipynb notebook."

**Comment by jbednar** [comment]
Brendan, here's what happened to the text I was looking for this morning; it does appear to have been lost in the earlier merge:

  https://github.com/bokeh/datashader/commit/a7a77f3a219955b7ec2525478d1f706e4a5c8910#commitcomment-16804515

**Comment by brendancol** [comment]
@jbednar ok making above changes, should i simply add the https://github.com/bokeh/datashader/commit/a7a77f3a219955b7ec2525478d1f706e4a5c8910#commitcomment-16804515 commit to this PR?

**Comment by jbednar** [comment]
Sure, or at least the changes from it (as the commit itself might have merge issues by now).

**Comment by jbednar** [comment]
Looks good, thanks!

**Comment by jbednar** [comment]
@brendancol, is this ready for me to merge?  Looks good to me, but I don't know if you're still adding more to it.

**Pull request merged**